src/_includes/banner.html

```diff
@@ -2,8 +2,9 @@
 {% comment %}Drop the div .alert classes to revert to the default banner styling.{% endcomment -%}
 <div class="banner alert alert-info">
   <p class="banner__text">
-    <a href="/dart-2"><b>Dart 2.0</b></a> is now available!
-    <a href="/install">Install Dart</a>.
+    <a href="https://medium.com/dartlang/dart-2-stable-and-the-dart-web-platform-3775d5f8eac7" target="_blank">Dart 2.0 is now stable!</a>
```

Comment:
- Consider adding `class="no-automatic-external"` and `rel="noopener"`?
- Also, how about putting **Dart 2** and/or **stable** in bold?

(We really should make this a markdown file, ... but in another PR.)

Reply:
I rather like having the external link, since it's otherwise hard to tell that the text has any links at all. `rel="noopener"` seems like a reasonable thing, though. (Do we ever automatically add that?)

Reply:
I'd like to generate those icons statically. At the same time adding `target="_blank" rel="noopener"` seems like an excellent idea.

Reply:
#1058.